CRM/Case/xml/Menu/Case.xml

```diff
@@ -136,4 +136,11 @@
     <path>civicrm/ajax/delcaserole</path>
     <page_callback>CRM_Case_Page_AJAX::deleteCaseRoles</page_callback>
   </item>
+  <item>
+    <path>civicrm/case/add-contact</path>
+    <page_callback>CRM_Case_Form_AddContact</page_callback>
+    <title>AddContact</title>
+    <access_arguments>access CiviCRM</access_arguments>
```

Comment:
I think access_arguments should be `access my cases and activities`

Reply:
Actually, what is this entry? I don't think this class exists. Can this be removed?